user-service/controller/user-controller.js

```diff
@@ -34,15 +38,79 @@ export async function signIn(req, res) {
         const { username, password } = req.body;
         const user = await _FindOneuser(username,password)
         if (user) {
-            const accessToken = jwt.sign({username:username},ACCESS_TOKEN_SECRET)
-            console.log(accessToken)
-            // res.json({accessToken:accessToken})
-            res.cookie("accesstoken",accessToken,{httpOnly:true})
-            return res.status(200).json({message: `Log in is successful!`,accesstoken:accessToken });
+            const accessToken = jwt.sign({username:username},ACCESS_TOKEN_SECRET,{expiresIn:"1h"})
+            const refreshToken = jwt.sign({username:username},REFRESH_ACCESS_TOKEN_SECRET,{expiresIn:"1h"})
+            user.refreshToken = refreshToken
+            user.save()
+            //refresh token in cookie
+            //access token in data
+            if(validatePassword(password, user.password)){
+                return res.cookie("refreshtoken",refreshToken,{httpOnly:true}).status(200).json({message: `Log in is successful!`,accesstoken:accessToken });

```

Comment:
Can you help split this into multiple lines to make it cleaner?

Reply:
Done!